src/Idris/DocString.idr

```diff
@@ -167,7 +191,18 @@ getDocsForName fc n
              let doc = show !(aliasName n) ++ " : " ++ show !(resugar [] ty)
                               ++ "\n" ++ addNL (indent str)
              extra <- getExtra n def
-             pure (doc ++ extra)
+             getInFixDocn <- getInFixDoc n
+             let infixes = case getInFixDocn of
+                    Just infixStr => infixStr
+                    Nothing       => ""
+             getPreFixDocn <- getPreFixDoc n
+             let prefixes = case getPreFixDocn of
+                    Just prefixStr => prefixStr
+                    Nothing        => ""
+             let fixes = if infixes == "" && prefixes == ""
+                            then ""
+                            else "\nFixity Declarations:\n" ++ infixes ++ prefixes
+             pure (doc ++ extra ++ fixes)
```

Comment:
Rather than adding additional newlines that are not needed if there is not extra / fixity
information, it would make more sense to use `unlines` here.

Reply:
I am afraid I do not know how to use `unlines` there, sorry…

Reply:
So should we also print the prefix version docs for `(-)` or just remove the prefix information?

Reply:
the trim NEWLINE was dealt with so former changes on tests was revert

to see what now it looks like at [add test](https://github.com/idris-lang/Idris2/compare/master...unsafe-coe:master#diff-359d888a70b3df58bad8867a5dfe6d7c65d5625b531e33085738c0ff1d0b73ad)